Application.mk

```diff
@@ -88,6 +88,7 @@ ifneq ($(BUILD_MODULE),y)
 endif
 
 ROOTDEPPATH += --dep-path .
```

Comment:
DEPPATH is used consitently in nuttx folder. Why not convert all ROOTDEPPATH to DEPPATH?

Reply:
Because I don't know why both ROOTDEPPATH and DEPPATH are existed. So just add it conveniently.
I'll be merged to DEPPATH in some apps if you think it is better.

Reply:
Yes, it's better to unify into DEPPATH, thanks.

Reply:
I added a new commit that unify into DEPPATH.